node_modules/helpers/modules/time/index.js

```diff
@@ -1,25 +1,69 @@
 function getTime() {
     const currentDate = new Date();
 
-    const hour = currentDate.getHours();
+    const hour_24 = currentDate.getHours();     //current hour in 24 hour format
+    const hour_12 = formatHour(hour_24);        //current hour in 12 hour format + AM/PM suffix
     const minutes = currentDate.getMinutes();
     const seconds = currentDate.getSeconds();
-    const time = `${hour}:${minutes}:${seconds}`;
+
+    const time = {
+        hour_24: hour_24,
+        hour_12: hour_12,
+        minutes: minutes,
+        seconds: seconds
+    };
 
     return time;
```

Comment:
We can further reduce this function to gain (minor) performance improvements; the way to achieve this would be to create the return-object directly in the `return` statement. Furthermore, we can prevent memory build-up by avoiding the creation of variables (or in this case constants) that are only ever used a single time (for a direct return in this case).
```js
return {
    hour_24: currentDate.getHours(),
    hour_12: formatHour(currentDate.getHours()),
    minutes: currentDate.getMinutes(),
    seconds: currentDate.getSeconds()
};
```